components/dialog/dialog-mixin.js

```diff
@@ -46,6 +46,8 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 		this._nestedShowing = false;
 		this._state = null;
 		this._left = 0;
+		this._overflowBottom = false;
```

Comment:
These are simply to fix the lit-analyzer errors that started cropping up. No functional difference here.